iotbx/cif/validation.py

```diff
@@ -369,7 +369,10 @@ def validate_loop(self, loop, block):
         if isinstance(references, string_types):
           references = [references]
         for reference in references:
-          ref_data = self.get_definition(reference)
+          try:
+            ref_data = self.get_definition(reference)
+          except:
```

Comment:
```suggestion
          except KeyError:
```
Based on the `get_defintion` and `find_definition` functions, it looks like the expected exception is a `KeyError`. Trapping the specific exception is helpful in case another part of the com fails. @olegsobolev and @nwmoriarty can double-check. This change should fix the failing Checks tests.